papermill/execute.py

```diff
@@ -136,7 +136,7 @@ def log_outputs(cell):
 
 
 def execute_notebook(notebook,
-                     output,
+                     output=None,
```

Comment:
Need a spec addition for this behavior :)

Reply:
Hi @jarrekk, I'm a bit confused why we would have output as None for the default? In general, output will need to go somewhere, and it's likely simpler to be explicit about where the output goes. Would you be able to give some additional context to specifically how you see this working? Thanks!